src/main/java/net/sf/jabref/external/ExternalFileTypes.java

```diff
@@ -184,7 +170,7 @@ public ExternalFileType getExternalFileTypeForName(String filename) {
                 foundType = type;
```

Comment:
Hmm, I must have missed that. (Or it was a method only called once with a constant input, do not remember fully.)